examples/win32-front-proxy/docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: "3.7"
+services:
+
+  front-envoy:
+    build:
+      context: .
+      dockerfile: Dockerfile-frontenvoy
+    networks:
+      - envoymesh
+    ports:
+      - "8080:8080"
+      - "8443:8443"
+      - "8003:8003"
+
+  service1:
+    build:
+      context: .
+      dockerfile: Dockerfile-service
+    # volumes:
+    #   - ./service-envoy.yaml:'./service-envoy.yaml'
+    networks:
+      - envoymesh
+    environment:
+      - SERVICE_NAME=1
+
+  service2:
+    build:
+      context: .
+      dockerfile: Dockerfile-service
+    # volumes:
```

Comment:
removable?

Reply:
fixed in ec182f94450b4b932ee74abcf989b127dc690666